framework_tvb/tvb/adapters/exporters/tvb_linked_export.py

```diff
@@ -55,13 +55,13 @@ def get_supported_types(self):
     def get_label(self):
         return "TVB Format with links"
 
-    def export(self, data, data_export_folder, project):
+    def export(self, data, export_folder, project):
         """
         Exports data type:
         1. If data is a normal data type, simply exports storage file (HDF format)
         2. If data is a DataTypeGroup creates a zip with all files for all data types
         """
-        self.copy_dt_to_export_folder(data, data_export_folder)
+        data_export_folder = self.copy_dt_to_export_folder(data, export_folder)
```

Comment:
as discussed, the multiple calls to storage_interface methods could have been merged into a single export_datatypes() method. This results in the possibility to exclude from the storage API some methods like write_zip_folder.
I am pushing these changes soon.